config/services.yaml

```diff
@@ -250,3 +251,8 @@ services:
     App\Service\OS2Forms\SubmissionNormalizers\HearingResponseSubmissionNormalizer:
         arguments:
             $selvbetjeningUserApiToken: '%env(SELVBETJENING_USER_API_TOKEN)%'
+
+    App\Controller\CaseEventController:
+        arguments:
+            $options:
+              view_timezone: '%view_timezone%'
```

Comment:
Any specific reason for not indenting this line with 4 spaces? Maybe we're missing an EditorConfig file (https://editorconfig.org/).

Reply:
Updated existing `.editorconfig` to indent `.yaml` with 4 spaces.